portfolio/src/main/webapp/index.html

```diff
@@ -26,7 +26,7 @@ <h1>Isabelle's Portfolio</h1>
       <form action="/data" method="POST">
         <h3>Add Your Comments:</h3>
         <textarea name="comments-input"></textarea>
-        <br></br>
+        <p></p>
         <input type="submit" />
```

Comment:
nit: I actually meant

```html
<p><textarea ...></textarea></p>
<p><input ...></p>
```

(also, it's better to address feedback on the original PR branch instead of in a follow-up PR, so that things don't look out-of-place wrt. the PR description)

Reply:
Oh okay, got it!
And I'll make sure to do that from here on out for PRs.